README.md

```diff
@@ -19,6 +19,22 @@ The Charts in the master branch of this repository match the latest packaged Cha
 
 The purpose of this repository is to provide a place for maintaining and contributing official Charts, with CI processes in place for managing the releasing of Charts into the Chart Repository.
 
+The Charts in this repository are organized into two folders:
+* stable
+* incubator
+
+Stable Charts:
+
+1. Provide a method for data persistence
+1. Support application upgrades
+1. Allow customization of the application configuration
+1. Provide a secure default configuration
+1. Do not leverage alpha features of Kubernetes
```

Comment:
Im going to leave it as is for now. I could see a case where a chart bundled the entirety of what was needed in dependent charts that made up the 3rd party extension